nimble/controller/src/ble_ll_conn.c

```diff
@@ -1038,8 +1038,19 @@ ble_ll_conn_adjust_pyld_len(struct ble_ll_conn_sm *connsm, uint16_t pyld_len)
     uint16_t ret;
 
 #if (BLE_LL_BT5_PHY_SUPPORTED == 1)
+    uint8_t phy_mode;
+
+    if (connsm->phy_tx_transition != BLE_PHY_TRANSITION_INVALID) {
+        phy_mode = ble_ll_phy_to_phy_mode(connsm->phy_tx_transition,
+                                          connsm->phy_data.phy_options);
+    } else {
+        phy_mode = ble_ll_phy_to_phy_mode(connsm->phy_data.tx_phy_mode,
```

Comment:
this does not seem right since `tx_phy_mode` already stores PHY mode so it should not be used as input to `ble_ll_phy_to_phy_mode`

Reply:
it is not right for sure. It should be phy_mode = connsm->phy_data.tx_phy_mode
will fix